package.json

```diff
@@ -51,6 +51,7 @@
     "trim-stats": "node -r esm bin-src/trim-stats-file.js",
     "storybook": "start-storybook -p 9009 -s static",
     "test": "jest",
+    "test:file": "jest -t",
```

Comment:
If I understand this correctly, you're using this command to run a specific test? Like `yarn test:file trim`.
If that's the case, I believe you can delete this line. `yarn test trim` would work as well for that use case